build.gradle

```diff
@@ -131,7 +131,6 @@ allprojects  {
             dependency 'jakarta.jms:jakarta.jms-api:2.0.3'
             dependency 'jakarta.xml.bind:jakarta.xml.bind-api:2.3.3'
             dependency 'org.glassfish.jaxb:jaxb-runtime:2.3.5'
-            dependency 'jakarta.validation:jakarta.validation-api:3.0.0'
```

Comment:
@ptuomola Any reason why we would replace the newer Jakarta validation lib with the older javax.* stuff? This dependency results in a couple of import statement changes further down... unless there is no strict requirement by Jersey 2.x I would revert this change.

Reply:
I tried first retaining the validation-api 3.0.0. The problem is that it is incompatible with about half a dozen libraries included in spring-boot-starter-jersey, and for many of which no version exists that is compatible for 3.0.0. So I had to backtrack and downgrade validation API to 2.x. 

I think overall it would make sense for us to try to remove as many of the specific library dependencies and instead use the Spring Boot bundles wherever they are available - that way we don't have to worry about compatibility of every individual library with everything else.